FV3/coupler_main.F90

```diff
@@ -135,7 +135,7 @@ program coupler_main
  call constants_init
  call sat_vapor_pres_init
  !$ser verbatim  call mpi_comm_rank(MPI_COMM_WORLD, mpi_rank,ier)
- !$ser init directory='.' prefix='Generator' mpi_rank=mpi_rank unique_id=.true.
+ !$ser init directory='../test_data/' prefix='Generator' mpi_rank=mpi_rank unique_id=.true.
```

Comment:
What's the reason to save the test data here instead?

Reply:
This was done to make it easier to manage the serialized data. When we save it for testing, I don't want to bring along all the model input and output files, as all the python model needs to run the tests is the serialized test data and the input.nml. We changed where we are storing the data (moved from docker volumes to buckets in the GCR), and I wanted to avoid either moving or deleting sets of the files before syncing to the bucket. Having the serializer write out to a different directory makes that easier. I put it one level up so my docker volumes were distinct, but then I probably made it less clear for the tests, since the test_data volume is inside rundir on the host :P. Maybe I'll change that